File: Swift/基本语法/第二部分/06-block的回顾/06-block的回顾/ViewController.m
//
//  ViewController.m
//  06-block的回顾
//
//  Created by tzsoft on 2018/2/23.
//  Copyright © 2018年 胡小羊. All rights reserved.
//

#import "ViewController.h"
#import "HttpTool.h"

@interface ViewController ()

@property (nonatomic ,strong) HttpTool *tools;
@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.tools = [[HttpTool alloc]init];
}
- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event{
    [self.tools loadData:^(NSString *jsonData) {
        NSLog(@"jsonData");
    }];
}

@end

File: Swift/基本语法/第二部分/12-HLYWeibo/09-AFNTools/09-AFNTools/ViewController.swift
//
//  ViewController.swift
//  09-AFNTools
//
//  Created by tzsoft on 2018/3/22.
//  Copyright © 2018年 胡小羊. All rights reserved.
//

import UIKit
import Alamofire

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()

    }
    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        NetworkTools.shareInstance.getRequest(url: "http://httpbin.org/get", requestType: .get,parameters: ["name":"huluyang"], success: { (respongObj) in
            print(respongObj!)
            let dataDict = respongObj
            print(dataDict!["url"]!)
            
        }) { (error) in
            print(error)
        }
    }
}

